docs/ManualDeploymentGuideScalarDLonAWS.md

```diff
@@ -0,0 +1,233 @@
+# Deploy Scalar DL on AWS
+
+Scalar DL is a database-agnostic distributed ledger middleware containerized with Docker.
+It can be deployed on various platforms and is recommended to be deployed on managed services for production to achieve high availability and scalability, and maintainability.
+This guide shows you how to manually deploy Scalar DL on a managed database service and a managed Kubernetes service in Amazon Web Services (AWS) as a starting point for deploying Scalar DL for production.
+
+## What we create
+
+![image](images/network_diagram_eks.png)
+
+In this guide, we will create the following components.
+
+* A VPC with NAT gateway
+* An EKS cluster with two Kubernetes node groups
+* A managed database service
+    * DynamoDB
+* A Bastion instance with a public IP
+* Amazon CloudWatch
+
```

Comment:
Do we need  to write the prerequisites for permissions as azure ?
https://github.com/scalar-labs/scalar-kubernetes/blob/master/docs/ManualDeploymentGuideScalarDLOnAzure.md#prerequisites

Reply:
Yes, I will create and update.